src/components/LanguageSettings.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import danSelected from "./icons/dan-selected.png";
+import danDeselected from "./icons/dan-deselected.png";
+import engSelected from "./icons/eng-selected.png";
+import engDeselected from "./icons/eng-deselected.png";
+
+import strings from "../i18n/definitions";
+
+import styled from "styled-components";
```

Comment:
Line 9-16 should be in its own file LanguageSettings.sc.js to be consistent with the rest of the repo.

Reply:
Added now as rule no. 4 in the Coding Conventions section: https://github.com/zeeguu-ecosystem/zeeguu-react/blob/development/README.md#code-conventions

Reply:
Of course it should